source/reduce/reducer.cpp

```diff
@@ -54,10 +54,10 @@ void Reducer::SetInterestingnessFunction(
 }
 
 Reducer::ReductionResultStatus Reducer::Run(
-    std::vector<uint32_t>&& binary_in, std::vector<uint32_t>* binary_out,
+    const std::vector<uint32_t>& binary_in, std::vector<uint32_t>* binary_out,
     spv_const_reducer_options options,
     spv_validator_options validator_options) {
-  std::vector<uint32_t> current_binary(std::move(binary_in));
+  std::vector<uint32_t> current_binary(binary_in);
```

Comment:
Similar to my previous comment: I don't think we should see these changes. Please try squashing your changes and/or rebasing. These commits should not be in your PR.

Reply:
Resolved conflicts.